composer.json

```diff
@@ -33,12 +41,16 @@
         }
     },
     "scripts": {
-        "test": "vendor/bin/phpspec run",
+        "test": [
+            "vendor/bin/phpspec run",
+            "vendor/bin/simple-phpunit --group NothingInstalled"
+        ],
         "test-ci": "vendor/bin/phpspec run -c phpspec.ci.yml"
     },
     "extra": {
+        "class": "Http\\Discovery\\Composer\\Plugin",
         "branch-alias": {
-            "dev-master": "1.9-dev"
+            "dev-master": "1.15-dev"
```

Comment:
i think we should change the branch to be 1.x and get rid of branch alias.

i will extract your general cleanup into a separate MR and merge that already to keep this separate.